src/WordPress/Hooks.php

```diff
@@ -185,14 +202,8 @@ public function purgeCacheByRelevantURLs($postId)
                 $urls = array_filter($urls, array($this, "urlIsHTTPS"));
             }
 
-            // Don't attempt to purge anything outside of the provided zone.
-            foreach ($urls as $key => $url) {
-                if (!Utils::strEndsWith(parse_url($url, PHP_URL_HOST), $wpDomain)) {
-                    unset($urls[$key]);
-                }
-            }
-
-            if (isset($zoneTag) && !empty($urls)) {
+            if (!empty($urls)) {
+                do_action('cloudflare_purge_by_urls', $urls, $postIds);
```

Comment:
what is the thinking here? i don't think we want this here. by this point, we have all the URLs we care about and should just be processing them in chunks.

Reply:
### that one is an action, not a filter. would be very useful here to use to clear local caches for given urls or completely considering CF will soon be requesting then again from origin

Reply:
perhaps a more appropriate name in that case would be something like `cloudflare_purged_urls` to denote it is URLs we are purging?